Use config mode to prevent conflict with /usr/lib64/cmake/Qt6/FindSnappy.cmake (qtwebengine)
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 34123b3..eef8a38 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -535,7 +535,7 @@ include_directories (${CMAKE_CURRENT_SOURCE_DIR}/compat)
 if (NOT WIN32 AND NOT ENABLE_STATIC_EXE)
 
     if (NOT ENABLE_STATIC_SNAPPY)
-        find_package (Snappy)
+        find_package (Snappy CONFIG)
     endif ()
 
     # zlib 1.2.4-1.2.5 made it impossible to read the last block of incomplete
